examples/vector_databases/Using_vector_databases_for_embeddings_search.ipynb

```diff
@@ -101,16 +104,16 @@
     "# Redis client library for Python\n",
     "import redis\n",
     "\n",
-    "# Pinecone's client library for Python\n",
+    "# Chroma's client library for Python\n",
```

Comment:
Here it looks like Chroma is just single commented, but the others are double commented - should they all be single commented?

Reply:
oops, fixed.